src/components/service/ServiceCard.tsx

```diff
@@ -72,19 +91,88 @@ const ServiceCard = ({ serviceData, isDiffMode, preState, state }: ServiceCardPr
               <code className="px-1 py-0.5 rounded text-xs font-mono bg-blue-100 dark:bg-blue-900/60 text-blue-800 dark:text-blue-200">
                 aₛ
               </code>
-              Storage
+              <span className="flex items-center gap-1">
+                <span>Storage</span>
+                {!isDiffMode ? (
+                  <span className="text-xs text-muted-foreground">({counts.storage.totalCount})</span>
+                ) : (
+                  <span className="text-xs text-muted-foreground">
+                    {counts.storage.changed > 0 || counts.storage.preCount !== counts.storage.postCount ? (
+                      <>
+                        ({counts.storage.preCount}/{counts.storage.postCount}
+                        {counts.storage.added > 0 && <span className="text-green-700 dark:text-green-400"> +{counts.storage.added}</span>}
+                        {counts.storage.removed > 0 && <span className="text-red-700 dark:text-red-400"> /-{counts.storage.removed}</span>}
+                        )
+                      </>
+                    ) : (
+                      <>
+                        ({counts.storage.postCount}
```

Comment:
```suggestion
                        ({counts.storage.preCount}
```
since the `+/-` information is next to it, so `postCount` is redundant, because it can be computed. Similarly in other tabs.